bin/stacks/routing-api-stack.ts

```diff
@@ -326,7 +327,8 @@ export class RoutingAPIStack extends cdk.Stack {
         },
       }),
       threshold: 20,
-      evaluationPeriods: 2,
+      evaluationPeriods: 5,
+      datapointsToAlarm: 4,
```

Comment:
SEV3 should probably be lower? 3 out of 5?

Reply:
changed it to 3 out of 5